xbmc/settings/AdvancedSettings.cpp

```diff
@@ -1086,6 +1087,7 @@ void CAdvancedSettings::ParseSettingsFile(const std::string &file)
   XMLUtils::GetUInt(pRootElement, "imageres", m_imageRes, 0, 9999);
   if (XMLUtils::GetString(pRootElement, "imagescalingalgorithm", tmp))
     m_imageScalingAlgorithm = CPictureScalingAlgorithm::FromString(tmp);
+  XMLUtils::GetUInt(pRootElement, "thumbqualityjpeg", m_thumbQualityJpeg, 0, 21);
```

Comment:
Mind changing thumb to art or image? Might look peeky but thumb/thumbnail has a long history in Kodi :)
By now thumb is just another generic art type